src/Microsoft.ML.CpuMath/Thunk.cs

```diff
@@ -22,7 +22,7 @@ internal static unsafe class Thunk
         public static extern void MatMulX(/*const*/ float* pmat, /*const*/ float* psrc, float* pdst, int crow, int ccol);
 
         [DllImport(NativePath), SuppressUnmanagedCodeSecurity]
-        public static extern void MatMulPA(/*const*/ float* pmat, /*const*/ int* pposSrc, /*const*/ float* psrc,
+        public static extern void MatMulP(/*const*/ float* pmat, /*const*/ int* pposSrc, /*const*/ float* psrc,
             int posMin, int iposMin, int iposLim, float* pdst, int crow, int ccol);
         [DllImport(NativePath), SuppressUnmanagedCodeSecurity]
         public static extern void MatMulPX(/*const*/ float* pmat, /*const*/ int* pposSrc, /*const*/ float* psrc,
```

Comment:
Who calls `MatMulPX` anymore?

Reply:
I also don't see it in `Sse.cpp`.  This should be removed.

---
In reply to: [225949263](https://github.com/dotnet/machinelearning/pull/1274#discussion_r225949263) [](ancestors = 225949263)

Reply:
i think its called in ```Avx.cs```. but i will remove it because ```Avx.cs```  file is not used anywhere

Reply:
Yes, please remove it from `Avx.cs` as well.